app/locale.js

```diff
@@ -147,14 +180,13 @@ exports.init = (language, cb) => {
   cb = cb || function () {}
 
   // Currently selected language identifier I.e. 'en-US'
-  lang = language
+  lang = language || defaultLocale()
 
   // Languages to support - TODO retrieve this dynamically
   const langs = [
     { code: 'en-US' },
     { code: 'nl-NL' },
-    { code: 'pt-BR' },
-    { code: 'bn-BD' }
+    { code: 'pt-BR' }
```

Comment:
Even within the same file we have 2 of the same list of languages, please calculate one from the other.

Please merge directly after that change and with the other localization merge that was backed out. Thanks!